frontend/src/domain/shared_slices/BeaconMalfunction.ts

```diff
@@ -144,22 +135,42 @@ const beaconMalfunctionSlice = createSlice({
 
     /**
      * Update a single beacon malfunction in the kanban
-     * @function updateLocalBeaconMalfunctions
-     * @memberOf BeaconMalfunctionReducer
+     *
      * @param {Object=} state
      * @param {{payload: BeaconMalfunction}} action - the beacon malfunction to update
      */
-    updateLocalBeaconMalfunctions(state, action) {
+    updateLocalBeaconMalfunction(state, action) {
       const id = action.payload?.id
       const nextBeaconMalfunctions = state.beaconMalfunctions.filter(beaconMalfunction => beaconMalfunction.id !== id)
 
       state.beaconMalfunctions = [action.payload, ...nextBeaconMalfunctions]
     },
 
+    /**
+     * Update a beacon malfunctions in the kanban
+     *
+     * @param {Object=} state
+     * @param {{payload: }} action - the beacon malfunctions to update
+     */
+    updateLocalBeaconMalfunctions(
+      state,
+      action: PayloadAction<{
+        beaconMalfunctions: BeaconMalfunction[]
+        ids: number[]
```

Comment:
This `ids` prop is a strategy to remove beaconsMalfunctions? Because if that's the case, I would rather use another way.

Reply:
I just realized that I can remove the `ids` param as the id is included in the `BeaconMalfunction` object.
But you were thinking of another way ?